front_end/package.json

```diff
@@ -4,12 +4,19 @@
   "description": "",
   "main": "main.js",
   "scripts": {
-    "start": "electron ."
+    "start": "electron .",
+    "test": "jest"
   },
   "keywords": [],
   "author": "me",
   "license": "ISC",
   "devDependencies": {
-    "electron": "^12.0.2"
+    "electron": "^12.0.2",
+    "jest": "^26.6.3",
+    "spectron": "^14.0.0"
```

Comment:
I suggest we leave these dependencies (line 8 too) out for the time being, at least until we implement some actual test cases.